metadata-db-app/src/cmr/metadata_db/data/oracle/concepts/subscription.clj

```diff
@@ -45,3 +46,9 @@
 (defmethod concepts/concept->insert-args [:subscription true]
   [concept _]
   (subscription-concept->insert-args concept))
+
+(defmethod concepts/after-save :subscription
+  [db provider sub]
+  (when (:deleted sub)
+    ;; Cascade deletion to real deletes of granules
```

Comment:
real deletes of subscription-notification

Add the business logic to this comment here